frontend/cypress/tests/codeView/codeView_filterGlob.js

```diff
@@ -0,0 +1,177 @@
+describe('filter glob', () => {
+  it('check filter glob radio button is clickable', () => {
+    // open the code panel
+    cy.get('.icon-button.fa-code')
+        .should('be.visible')
+        .first()
+        .click();
+
+    // click on filter glob radio button
+    cy.get('.radio-button--search')
+        .should('be.visible')
+        .click();
+
+    // filter by file type should be unchecked
+    cy.get('.radio-button--checkbox')
+        .should('not.be.checked');
+  });
+
+  it('check no filter glob input should show all file types by default', () => {
+    cy.get('.icon-button.fa-code')
+        .should('be.visible')
+        .first()
+        .click();
+
+    cy.get('.radio-button--search')
+        .should('be.visible')
+        .click();
+
+    // the contents of all the files should be visible
+    cy.get('#tab-authorship .files', { timeout: 90000 })
```

Comment:
For this, it is time to wait before the `get` timeout. I think it is ok to remove it as well. But I also thought that we could include the `timeout` option in case the loading of content takes some time to get resolved. `90000` is consistent with other previous tests that try to load files at `#tab-authorship` as well. Feel free to let me know what you think :)